5calls/app/src/main/java/org/a5calls/android/a5calls/controller/RepCallActivity.java

```diff
@@ -69,6 +69,7 @@
     private Tracker mTracker = null;
 
     @BindView(R.id.scroll_view) ScrollView scrollView;
+    @BindView(R.id.rep_call_root) LinearLayout linearLayout;
```

Comment:
Give this a better name -- linearLayout is too generic.

Reply:
Removed.